rfcs/text/0000-conceptual-data-model.md

```diff
@@ -0,0 +1,111 @@
+
+The providence graph gives the exact history of data and metadata as it existed at the time a given process was run.  This view provides the ability to analyze and understand an experiment.  It is a reverse traversal from leaves to roots following FQIDs of the input to each metadata instance as it existed at the time the leaf was created.  Since metadata indirectly upstream of a process may change without intermediary processes being rerun, the providence links are not simply the version of the direct input to a node at the time each metadata entity was created.  The upstream metadata change may have triggered a downstream rerun without intermediate processes being rerun.
+
+The knowledge if an update will result in a new or amended analysis is not known by the ingestion process.  It will be determined by each analysis examining the update.
+
+Figures 1-4 are some examples to illustrate the graph interpretation issues.  In each of these drawings, the graph node labels starting with B are biomaterials, those with P are processes, and with F are files.  The first number is the unique id, which is a UUID in practice.  The number after the dot is the version, which is represented as a timestamp.  Red squiggles indicate an outdated metadata instance.
+
+
+![](../images/0000-basic-graph.png)
+
+Figure 1: Basic graph of sequencing and analysis.  This is a basic experimental pipeline with no updates. The head graph is obtained by traversing the graph from the root to the leaves.  The provenance of a given leaf is achieved by reverse traversing the edges.
+
+
+![](../images/0000-updated-analysis-graph.png)
+
+Figure 2: Graph with an updated analysis.  In this case, an analysis is rerun for a reason other than an upstream change.  For example, this could be due to a bug be discovered in the analysis software.  The head graph is obtained by traversing the graph, ignoring the outdated nodes.  The provenance graph is obtained by reverse traversing the edges shown in red.
+
+![](../images/0000-new-analysis-graph.png)
+
+Figure 3: Graph with a new analysis:  In this case, a second, independent analysis is run.  This might be the case when a new different analysis algorithm is being used.  The head graph includes both analysis results.   The provenance graph is obtained by reverse traversing the edges shown in red.
+
+
+![](../images/0000-reanalysis-graph.png)
+
+Figure 4: Graph with an upstream consequential metadata change that triggers a reanalysis.  This is the case where there is an error in a metadata field that is used to parameterize the analysis. Here the sequencing didn't change, however, the downstream analysis was triggered by a change in the biomaterial metadata (B1).  The provenance graph reverse traverses the edges shown in red, going to B1.2 even though the process metadata P1.1 has not changed.
```

Comment:
Is one issue here how to mark the old analysis as deprecated without having to create a downstream update just to record the deprecation. Is it acceptable to mark deprecation in the corrected bundle (I'm anticipating not) or perhaps have some bundle headers (meta-metadata) that record deprecation?